src/esp/metadata/attributes/AttributesEnumMaps.h

```diff
@@ -17,6 +17,45 @@ enum class MotionType;
 namespace metadata {
 namespace attributes {
 
+/**
+ * @brief This enum class defines the possible options for what will be rendered
+ * for a particular articulated object.
+ *
+ */
+enum class ArticulatedObjectRenderMode {
+  /**
+   * Represents the user not specifying which rendering mode to use. Resorts
+   * to any previously known/set value
+   */
+  Unspecified = ID_UNDEFINED,
+  /**
+   * Render the articulated object using its skin if it has one, otherwise
+   * render it using primitives.
+   */
+  Default,
+  /**
+   * Render the articulated object using its skin.
+   */
+  Skin,
+  /**
+   * Render the articulated object using primitives to respresent each link.
+   */
+  Primitives,
```

Comment:
Let's call this `LinkVisuals` to be explicit that this means rendering the `<visual>` tag in the `<link>` which is typically a `mesh` type.